source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,32 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
+that matches the filter.
+
+Create an Options object to specify additional options. Specify the
+``upsert`` option to create a new document if no documents match the
+filter.  To receive acknowledgment in the response from MongoDB that the
+write operation was performed on the specified number of :manual:`mongod
+</reference/program/mongod>` servers, set the ``w`` option. 
+
+The ``updateOne()`` method returns a document containing information such
+as the number of documents that match the filter, and the number of
+documents that were modified.
+
+Specifying incorrect parameters for your ``updateOne()`` operation can
```

Comment:
This sentence sounds like we're going to discuss multiple error cases, but we only talk about one. I feel like we might just not need the first sentence here at all. What do you think?

Reply:
I disagree, the first line exists as an introduction to the paragraph that informs the user there can be user errors that occur. It's a bridge from the return paragraph to the error paragraph

Reply:
Good point -- a bridge is definitely necessary here! I think if we want to build a more comprehensive paragraph describing errors it might be a good idea to make sure that this bridge introduces all potential errors, and not just the subset of them that are caused by incorrect parameters (permissions errors, broken connection errors, timeouts, etc. all fall outside of this category). It might be tricky to do this without sounding too vague, but how about something like:
```
``updateOne()`` throws an error in several exceptional
circumstances. For instance...
```

Could even throw a bulleted list under there. Only thing I can't figure out right now is how to explain how the callback version of the method doesn't throw exceptions :)